src/base/_forms.css

```diff
@@ -52,6 +52,19 @@ details > summary,
   appearance: none;
 }
 
+.outline {
+  background: transparent;
+  box-shadow: none;
+  color: var(--form-text);
+  border: 2px solid var(--element-bg);
+}
+
+.outline:hover {
```

Comment:
Maybe also include the `:focus` state too

Reply:
Good point, added